content/en/docs/concepts/workloads/pods/init-containers.md

```diff
@@ -289,6 +289,106 @@ The Pod which is already running correctly would be killed by `activeDeadlineSec
 The name of each app and init container in a Pod must be unique; a
 validation error is thrown for any container sharing a name with another.
 
+### Sidecar containers and restartPolicy
+
+{{< feature-state for_k8s_version="v1.28" state="alpha" >}}
+
+Thanks to a newly available feature gate named `SidecarContainers`, each
+init container can have its own `restartPolicy`, independent of the Pod and the
+other init containers, as well as any regular [probe](/docs/concepts/workloads/pods/pod-lifecycle/#types-of-probe).
+
+If an init container is created with a `restartPolicy` set to `Always`, it will
+start and remain running during the entire life of the Pod, which is useful for
+running supporting services isolated from the main application container.
+You can also specify a `startupProbe` and/or a `readinessProbe` to help kubelet
+determine the `ready` state of these containers.
```

Comment:
You can remove `startupProbe` here and add it to L312.

Reply:
thanks for the feedback, updated